spec/spec_helper.rb

```diff
@@ -3,6 +3,12 @@
 require 'simplecov'
 require 'coveralls'
 
+# This works around the hash extensions not being automatically included in ActiveSupport < 4
+require 'active_support/version'
+require 'active_support/core_ext/hash' if ActiveSupport::VERSION &&
+                                          ActiveSupport::VERSION::MAJOR &&
+                                          ActiveSupport::VERSION::MAJOR < 4
```

Comment:
is `ActiveSupport.version.to_s < '4'` the same?

Reply:
ActiveSupport 3 has a different API for version (https://github.com/rails/rails/blob/3-2-stable/activesupport/lib/active_support/version.rb). It's closer to 'activesupport/gem_version' in 4+.